ext/TensorKitChainRulesCoreExt/factorizations.jl

```diff
@@ -23,12 +24,11 @@ function ChainRulesCore.rrule(::typeof(TensorKit.tsvd!), t::AbstractTensorMap;
     function tsvd!_pullback(ΔUSVϵ)
         ΔU, ΔΣ, ΔV⁺, = unthunk.(ΔUSVϵ)
         Δt = similar(t)
-        for (c, b) in blocks(Δt)
-            Uc, Σc, V⁺c = block(U, c), block(Σ, c), block(V⁺, c)
-            ΔUc, ΔΣc, ΔV⁺c = block(ΔU, c), block(ΔΣ, c), block(ΔV⁺, c)
-            Σdc = view(Σc, diagind(Σc))
-            ΔΣdc = (ΔΣc isa AbstractZero) ? ΔΣc : view(ΔΣc, diagind(ΔΣc))
-            svd_pullback!(b, Uc, Σdc, V⁺c, ΔUc, ΔΣdc, ΔV⁺c)
+        foreachblock(Δt) do (c, b)
+            USVᴴc = block(U, c), block(Σ, c), block(V⁺, c)
+            ΔUSVᴴc = block(ΔU, c), block(ΔΣ, c), block(ΔV⁺, c)
```

Comment:
Not sure if the formatter agrees, but I kinda like explicit parenthesis around tuples.